src/main/java/jupyter/AsDisplayData.java

```diff
@@ -0,0 +1,19 @@
+package jupyter;
+
+import java.util.Map;
+
+/**
+ * Convenience interface for classes whose display data is known in advance.
+ *
+ * Classes implementing this interface are displayed via the {@link AsDisplayData#display} method
+ * by default, without needing to register a {@link Displayer} in advance.
+ */
+public interface AsDisplayData {
+    Map<String, String> display();
```

Comment:
Nit: The formatting of this file should match the rest of the project: 2 spaces to indent and max 100 character lines.

Reply:
Is there automated linting to apply for Java so that we can set Travis/bots to do the nits?

Reply:
Fixed.

Reply:
I'm sure there's a way to do it, but it isn't set up for this project. It isn't that difficult to note style issues.